cmd/thriftrw-list-deps/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+
+	"github.com/jessevdk/go-flags"
+
+	"go.uber.org/thriftrw/compile"
+)
+
+type args struct {
+	ThriftFile string `positional-arg-name:"thrift-file"`
```

Comment:
nit: I would call it just "File" (and the positional arg "FILE").

And can we add a `description:"Path to the Thrift file"`?